src/systems/ships/spawn_ships.rs

```diff
@@ -15,30 +14,7 @@ pub fn spawn_random_ships(mut commands: Commands, asset_server: Res<AssetServer>
             faction_starship: random(),
             engine: random(),
             velocity: 30.0,
-            rotation: f32::to_radians(10.0),
             size: Vec2 { x: 100.0, y: 100.0 },
-            acceleration: 0.1,
-            current_velocity: 0.0,
```

Comment:
Current Velocity, Rotation and Acceleration is for player movement so I have moved it out to there.

AI just go a constant speed and think that is probably the better way. We will need to flush out an in combat AI system at some point